### DISCUSSION TITLE
Edges not visible

### DISCUSSION THREAD

**Discussion by arkapratimc** [bcakmakoglu/vue-flow#1069] [Q&A] [answered]
Stackblitz link :- https://stackblitz.com/edit/github-85efq7?file=pages%2Findex.vue

I tried to make a custom node and attach the [built-in edges](https://vueflow.dev/guide/edge.html#default-edge-types) to it.

I can only see a black patch and nothing else. Will the community people kindly help me in solving this issue.

Thank you

**Comment by bcakmakoglu** [comment, marked as answer]
You're not importing the necessary styles.

```css
/* import the required styles */
@import 'node_modules/@vue-flow/core/dist/style.css';

/* import the default theme (optional) */
@import 'node_modules/@vue-flow/core/dist/theme-default.css';
```

**Reply by bcakmakoglu** [reply]
Another problem with your app:
The wrapping element of Vue Flow *must have a height*.

```vue
<div style="height: 100vh">
    <VueFlow v-model="elements">...
```

**Comment by arkapratimc** [comment]
Ah! tysm & very sorry for the late reply
